### DISCUSSION TITLE
webpack built express server is not finding the linked bundle.js file

### DISCUSSION THREAD

**Discussion by knightswatch3** [webpack/webpack#14324] [Q&A] [answered]
My webpack built express server is not able to find the bundle file. 
```The resource from “http://localhost:5000/main-bundle.js” was blocked due to MIME type (“text/html”) mismatch (X-Content-Type-Options: nosniff).```

Here is my https://github.com/SatishChandraPalakoti/svelte-1

None of the online suggestions are helpful. Please help TIA

**Comment by alexander-akait** [comment, marked as answer]
Something wrong with `publicPath`, try to remove ` publicPath: '/',`, default `auto` value is better to use
